### PR TITLE
128T 4.5.1 release notes

### PR DESCRIPTION
**Pull request by MichaelBaj**

### CODE REVIEW

File: docs/release_notes_128t_4.5.md [hunk 1]
```diff
@@ -3,6 +3,122 @@ title: 128T 4.5 Release Notes
...
+
+- **I95-33193** "About This System" within GUI now displays additional information specific to installation type (e.g., ISO vs. RPM)
+------
+- **I95-35319** "Managed Reports" (Custom Report factory defaults) are now shipped with the product.
+------
+- **I95-36112** Q-in-Q VLAN stacking
+
+### Issues Fixed
+
+- **I95-18807** Innocuous error produced in journal due to imudp module loaded by rsyslog daemon.
```
Comment (ptimmons) [inline docs/release_notes_128t_4.5.md:18]: "Removed a benign error displayed in journal..."

File: docs/release_notes_128t_4.5.md [hunk 2]
```diff
@@ -3,6 +3,122 @@ title: 128T 4.5 Release Notes
...
+
+  _**Symptoms:**_ The following message can be seen in the journal:
+  ```
+  rsyslogd[1337]: imudp: module loaded, but no listeners defined - no input will be gathered [v8.24.0 try http://www.rsyslog.com/e/2212 ]
+  ```
+------
+- **I95-33989** Incorrect error message reported within PCLI when trying to execute `validate` after a previous _validate_ was terminated with `CTRL+c`.
```
Comment (ptimmons) [inline docs/release_notes_128t_4.5.md:25]: Terminating a "validate" command with CTRL-c returns to the PCLI prompt but does not stop the in-progress validation. This prevents subsequent validation attempts until the in-progress validation completes in the background.

File: docs/release_notes_128t_4.5.md [hunk 3]
```diff
@@ -3,6 +3,122 @@ title: 128T 4.5 Release Notes
...
+
+  Until the system is upgraded to 4.5.1, this issue can be mitigated by configuring dns-config via the PCLI.
+------
+- **I95-36536** Deleting a session-capture filter within the PCLI can cause the highway process to fault.
+------
+- **I95-36537** Dynamic session-captures are now created with a default session count of 100 instead of unlimited.
+------
+- **I95-36564** Buffer queue depth allocation algorithm was inefficient causing latency in session setup.
+------
+- **I95-36574** After a HA interface fail over, a session collision can occur between the recovered flow and an existing reverse flow. The recovered flow does not get setup properly and can cause the highway process to fault upon session expiry.
```
Comment (ptimmons) [inline docs/release_notes_128t_4.5.md:73]: After an HA interface failover...
... does not get set up properly...

**Review by ptimmons — APPROVED** [review]

File: docs/release_notes_128t_4.5.md [hunk 4]
```diff
@@ -78,46 +92,61 @@ sidebar_label: 4.5
...
 ------
-- **I95-37006** Peer path establishment may fail for waypoint interfaces that use DHCP (e.g., LTE) when upgrading from 4.4.x
+- **I95-36927** Race condition can cause a fault in the highway process during session setup and configuration change removes BGP service route path
 ------
-- **I95-37042** 128T process `prank` journal logs was incorrectly excluded from output of `save tech-support-info`
+- **I95-37006** Peer path establishment may fail for waypoint interfaces that use DHCP (e.g., LTE) when upgrading from 4.4.x.
+------
+- **I95-37021, I95-37026** Configuring overlapping session-types could cause the highway process to ignore the configuration change.
+------
+- **I95-37042** 128T process `prank` journal logs was incorrectly excluded from output of `save tech-support-info`.
```
Comment (ptimmons) [inline docs/release_notes_128t_4.5.md:137]: were incorrectly

**Review by ptimmons — DISMISSED** [review]
One small nit.

**Review by jeffmart — APPROVED** [review]

File: docs/release_notes_128t_4.5.md (hunk 4 above)
Comment (jeffmart) [reply docs/release_notes_128t_4.5.md]: This has been addressed with Mikes update

**Pull request merged**